content/en/news/releases/1.14.x/announcing-1.14/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Announcing Istio 1.14
+linktitle: 1.14
+subtitle: Major Update
+description: Istio 1.14 release announcement.
+publishdate: 2022-05-11
+release: 1.14.0
+skip_list: true
+aliases:
+- /news/announcing-1.14
+- /news/announcing-1.14.0
+---
+
+We are pleased to announce the release of Istio 1.14!
+
+{{< relnote >}}
+
+This is the second Istio release of 2022. We would like to thank the entire Istio community
+for helping to get Istio 1.14.0 published.
+Special thanks are due to the release managers Lei Tang (Google) and Greg Hanson (solo.io),
+and to Test & Release WG lead Eric Van Norman (IBM) for his help and guidance.
+
+{{< tip >}}
+Istio 1.14.0 is officially supported on Kubernetes versions `1.20` to `1.23`.
+{{< /tip >}}
+
+Here are some of the highlights of the release:
+
+## Add a new approach for CA integration through the Envoy SDS API
+
+This new feature adds a new approach for CA integration through the Envoy SDS API.
+For more information, check out the [documentation](/docs/ops/integrations/spire/).
```

Comment:
The doc at /docs/ops/integrations/spire/ is in the PR https://github.com/istio/istio.io/pull/11057, which is currently under review.

Reply:
The lint test failure for the broken link to /docs/ops/integrations/spire/ is expected because https://github.com/istio/istio.io/pull/11057 has not been merged yet.

Reply:
Hey @lei-tang, just a friendly reminder that there is also supporting deployment example for this at https://github.com/istio/istio/pull/37948